cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -174,6 +175,27 @@ def _read_old_file_lines(self, file_path):
             lines = f.readlines()
         return lines
 
+    def _bytes_as_mb(self, value_in_bytes):
+        return '{}MB'.format(value_in_bytes / 1024 / 1024)
+
+    def _generate_default_shared_buffers(self):
+        return self._bytes_as_mb(psutil.virtual_memory().total / 4)
+
+    def _generate_default_effective_cache_size(self):
+        if config[POSTGRESQL_SERVER]['cluster']['nodes']:
```

Comment:
probably comment/docstring saying _why_ is it different for clustered or not

Reply:
Actually, this should be depending on whether the database_server is the only locally installed component, not on whether it's a cluster (because of single-db install mode).